app/components/lg-lion-search.js

```diff
@@ -54,9 +61,31 @@ export default Ember.Component.extend({
       params['name'] = selectedName;
     }
 
+    if (selectedDob && !Ember.isBlank(selectedDob)) {
+      var searchRange = this.get('searchRange');
+
+      var thisYear = (new Date()).getFullYear();
+      var earlyYear = new Date(selectedDob);
+      var lateYear = new Date(selectedDob);
+
+      earlyYear.setFullYear(earlyYear.getFullYear() - searchRange);
+      lateYear.setFullYear(lateYear.getFullYear() + searchRange);
+
+      if (earlyYear.getFullYear() < (thisYear - 30)) {
+        earlyYear.setFullYear(thisYear - 75); // all old lions
```

Comment:
@redyaffle  it would be nice to have this logic in the units file. having that searchRange=3 variable is a bit inflexible. I think this is fine for now to get something working, but a cleaner solution in the future would be to make the range explicitly in the units file, something like
`var = searchDobOptions = [{label:'5-10 years', value:{start: 5, end: 10}]`  This is another reason for a separate search component so that we can define the units as a pair of dates.